leaf-ui/js/loadSaveFunctions.js

```diff
@@ -198,21 +181,50 @@ function getFuncSegList(arr) {
 	return res;
 }
 
+*/
+
 /**
- * Returns an object that contains the current graph, model, and analysis request.
- * This return object is what the user would download when clicking the Save button
- * in the top menu bar.
+ * Returns a backbone model Actor with information from the obj
  *
- * @returns {Object}
  */
-function getModelJson() {
-	var obj = {};
-	obj.graph = graph.toJSON();
-	obj.model = model;
-	obj.analysisRequest = analysisRequest;
-	return obj;
+function createBBActor(cell){
+	var actor = cell.get('actor');
+	var actorbbm = new ActorBBM({type: actor.type, actorName: actor.attributes.actorName});
```

Comment:
Can we change the name of the var to maybe `actorBBM` ? All lowercase looks odd I think

Reply:
Sure